arkane/ess/gaussian.py

```diff
@@ -164,12 +164,18 @@ def load_geometry(self):
         Return the optimum geometry of the molecular configuration from the
         Gaussian log file. If multiple such geometries are identified, only the
         last is returned.
+        Also checks that the Cartesian coordinates are printed in the input orientation:
+        IOP(2/9=2000) must be specified for large cases (14+ atoms)
         """
         number, coord, mass = [], [], []
 
+        iop2_9_equals_2000 = False
         with open(self.path, 'r') as f:
             line = f.readline()
             while line != '':
+                if '2/9=2000' in line:
+                    iop2_9_equals_2000 = True
```

Comment:
This is the wrong place in the code for the check. It isn't about the Cartesian coordinates-- it's the fact that the Hessian isn't consistent with those Cartesian coordinates, so move this to the load_force_matrix function